src/lib.rs

```diff
@@ -977,6 +977,21 @@ pub fn version_is_supported(version: u32) -> bool {
     }
 }
 
+macro_rules! add_frame {
+    {$frames:expr, $frame:expr, $payload_len: expr, $left:expr} => {{
+        if $frame.wire_len() <= $left {
```

Comment:
It'd be easier to read inverting the condition:
```rust
if $frame.wire_len() > $left {
    return false;
}

<add frame here>

true
```

Reply:
I tried this but I got this problem: https://stackoverflow.com/questions/24502282/why-does-removing-return-give-me-an-error-expected-type-but-found-type
Also can't use `return` because it will return from the function (`send()`) itself.